cpp/lib/ome/bioformats/tiff/IFD.cpp

```diff
@@ -1057,7 +1090,11 @@ namespace ome
             break;
 
           default:
-            throw Exception("Unsupported OME data model PixelType");
+            {
+              boost::format fmt("Unsupported OME data model PixelType %1%");
```

Comment:
There's a "by" in the previous very-similar message; maybe the difference makes sense though. I'm happy with whatever looks correct to you!